doc/mgr/cephadm.rst

```diff
@@ -1,13 +1,13 @@
-================
-SSH orchestrator
-================
+====================
+cephadm orchestrator
+====================
 
-The SSH orchestrator is an orchestrator module that does not rely on a separate
+The cephadm orchestrator is an orchestrator module that does not rely on a separate
```

Comment:
Eventually we want to mention the CLI tool here as well.

Reply:
this is changed in a later commit in the series